### PR TITLE
link previous versions of ap items as rewrites

### PR DESCRIPTION
**Pull request by petrjasek**
to avoid resending media

SDCP-588

### CODE REVIEW

File: server/tests/ingest/parser/ap_test.py [hunk 1]
```diff
@@ -340,6 +347,7 @@ def test_aps_category(self):
     def format(self, item):
         with patch.dict(superdesk.resources, resources):
             item["unique_id"] = 1
+            print("item", item)
```
Comment (dev-tsoni) [inline server/tests/ingest/parser/ap_test.py:350]: print.

File: server/cp/ingest/parser/ap.py [hunk 2]
```diff
@@ -851,6 +859,10 @@ def _parse_ap_products(self, data, item):
             for p in products
         ])
 
+    def categorisation_mapping(self, in_item, item):
```
Comment (dev-tsoni) [inline server/cp/ingest/parser/ap.py:862]: Is this function used somewhere?

Comment (petrjasek) [reply server/cp/ingest/parser/ap.py:862]: it's called by the parent class doing some mapping which should not be used by cp

Comment (dev-tsoni) [reply server/cp/ingest/parser/ap.py:862]: okay.

**Review by dev-tsoni — APPROVED** [review]

**Pull request merged**